src/builders/browser/schema.d.ts

```diff
@@ -9,7 +9,7 @@
 import { NormalizedBrowserBuilderSchema } from '@angular-devkit/build-angular';
 
 export interface SingleSpaBuilderSchema extends NormalizedBrowserBuilderSchema {
-  libraryTarget: string;
+  libraryTarget: "var" | "assign" | "this" | "window" | "global" | "commonjs" | "commonjs2" | "amd" | "umd" | "jsonp" | undefined;
```

Comment:
This is a duplication of [this type in the webpack repo](https://github.com/webpack/webpack/blob/0c1beb99a82bdd8d262673b7c54d805135ffd91a/declarations/WebpackOptions.d.ts#L1111). I'm not an expert in typescript -- is there a way to use that definition instead of duplicate it?

See related https://github.com/webpack/webpack/pull/8864 which is going to add one more possible libraryTarget -- just an example of where it would be better to reuse/inherit webpack's definitive definition instead of duplicating our own.

Reply:
I have to admit, I'm no typescript expert either. From my understanding though, the typescript here is generated from schema.json. 

As an experiment, I changed the definition in the schema to use a $ref to the definition in webpack/schemas/WebpackOptions.json and then used [json-schema-to-typescript](https://www.npmjs.com/package/json-schema-to-typescript) to generate a new schema file: the output was the same as I committed. 

I'm not happy with the fact that the $ref was pointing to ~/../../../node_modules/webpack/... so I won't be adding that to the PR.

Reply:
cool we can just update this when that webpack PR gets merged in